python-backend/app/api/nris_services/NRIS_service.py

```diff
@@ -80,7 +83,7 @@ def _get_EMPR_data_from_NRIS(mine_no):
         try:
             empr_nris_resp.raise_for_status()
         except requests.exceptions.HTTPError:
-            #TODO add logging for this error.
+            # TODO add logging for this error.
```

Comment:
Was this missed or is this blocked by something?

Reply:
Not sure honestly.